radis/lbl/broadening.py

```diff
@@ -227,21 +214,25 @@ def gaussian_lineshape(w_centered, hwhm):
 
 
 def gaussian_FT(w_centered, hwhm):
-    """Fourier Transform of a Gaussian lineshape.
+    r"""Fourier Transform of a Gaussian lineshape.
+
+    .. math::
+        \operatorname{exp}\left(\frac{-\left({\left(2\pi w_{centered} hwhm\right)}^2\right)}{4\ln2}\right)
```

Comment:
@dcmvdbekerom  for information : computed from your Python expression with [py2tex](https://pytexit.readthedocs.io/). Quite useful to write docs !